### PR TITLE
Text changes

### PR DESCRIPTION
**Pull request by pcuk1979**
Tidying up content design.

### CODE REVIEW

File: lib/pages/register/content/index.js [hunk 1]
```diff
@@ -3,25 +3,25 @@ module.exports = {
...
     confirm: {
       label: 'Confirm your password'
     }
   },
   account: {
-    setup: 'Your account will be created using {{email}}'
+    setup: 'Your account will be created using {{email}}.'
   },
-  acceptInvitation: 'Click the button below to accept this invitation and to link your profile to this establishment',
+  acceptInvitation: 'Create a password to accept this invitation and link your profile to this establishment.',
```
Comment (lennym) [inline lib/pages/register/content/index.js]: I have a feeling that this same text is used for the case where a user is adding a second establishment, and so will already have a password set. Is that right @joefitter?

Comment (joefitter) [reply lib/pages/register/content/index.js]: this is specifically for the case where they already have a password set and are just linking to a new establishment

**Comment by lennym** [comment]
@pcuk1979 What's going on with this? Will you be making changes, or shall we close it?

**Comment by pcuk1979** [comment]
I'm trying to recreate this journey on https://public-ui.notprod.asl.homeoffice.gov.uk, so I can see what the page looks like. But I'm not sure I can actually do this.

@lennym Am I correct?

**Comment by lennym** [comment]
@pcuk1979 TBH I think it doesn't matter because this journey is going to change anyway.

However, if you're super keen then just invite yourself to both establishments when logged in with the usual username/password.

**Comment by pcuk1979** [comment]
Ah yes - as this is related to the changes in the journeys you were mentioning on Friday in Slack, I'd just close this for the time being as it is bound to pop up again.

**Comment by lennym** [comment]
Done.

**Pull request closed without merging**